var/spack/repos/builtin/packages/py-xdot/package.py

```diff
@@ -27,6 +27,7 @@ class PyXdot(PythonPackage):
     depends_on('py-pygobject', type=('build', 'run'))
     depends_on('py-pycairo', type=('build', 'run'))
     depends_on('pango', type=('build', 'run'))
+    depends_on('harfbuzz', type=('build', 'run'))
     depends_on('atk', type=('build', 'run'))
     depends_on('gdk-pixbuf', type=('build', 'run'))
     depends_on('gtkplus', type=('build', 'run'))
```

Comment:
Should these have a link dep? If the compiler uses them, then we should add a link dep. If it's only used post-install, then we can keep them build/run.

Reply:
xdot itself is pure python so it doesn't require linking to harfbuzz